multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
Hold on... I'm confused now. I thought you said we want to put all the templates at the top-level folder? @9gix